exonum/src/blockchain/mod.rs

```diff
@@ -153,6 +153,12 @@ impl Blockchain {
         self.db.merge(patch)
     }
 
+    /// Commits changes from the patch to the blockchain storage with fsync.
+    /// See [`Fork`](../storage/struct.Fork.html) for details.
```

Comment:
I would rather prefer to see [this link](https://docs.rs/exonum/0.7.0/exonum/storage/trait.Database.html#tymethod.merge_sync) here.